int32 motor
bool enabled